test/test.ts

```diff
@@ -862,59 +862,56 @@ function runTests(targetPlatform: platform.IPlatform, useWkWebView: boolean): vo
             }
         });
 
-        describe("#localPackage.installOnNextRestart2x", function() {
+        if (!onlyRunCoreTests) {
```

Comment:
Mocha has the ability to filter tests after name. If all core tests would contain a `core` prefix, you would not need to do this.